platform/mds/mds-web/src/main/resources/webapp/js/directives.js

```diff
@@ -1588,9 +1588,11 @@
                             if (noSelectedFields) {
                                 $('.page_' + target + '_center').hide();
                                 $('.ui-jqgrid-status-label').removeClass('hidden');
+                                $('#entityInstancesTable .ui-jqgrid-hdiv').hide();
```

Comment:
we should also handle trash/history grids. Do they have the same ID? If not, we should rather base this on a class perhaps